### PR TITLE
Replace Travis with GHA

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Comment by ErikSchierboom** [comment]
Closes https://github.com/exercism/delphi/issues/417

**Review by rpottsoh — APPROVED** [review]

**Comment by rpottsoh** [comment]
Thanks @ErikSchierboom

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Exercism Delphi Pascal Track
 
-[![Build Status](https://travis-ci.org/exercism/delphi.svg?branch=master)](https://travis-ci.org/exercism/delphi)
+![CI](https://github.com/exercism/delphi/workflows/configlet/badge.svg)
```
Comment (rpottsoh) [inline README.md:3]: @ErikSchierboom, is `/workflows/configlet/badge.svg` something that is coming?

Comment (ErikSchierboom) [reply README.md:3]: configlet is still using Travis at the moment, but I've [created an issue](https://github.com/exercism/configlet/issues/184) to get it converted to GitHub actions :)

**Pull request merged**